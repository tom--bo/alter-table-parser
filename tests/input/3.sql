alter table t1 add column c45 varchar(45) after c44, algorithm inplace;
