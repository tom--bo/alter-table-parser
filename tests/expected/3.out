Input Table Definition: (Please type ^d to end input) 
type = 1
type = 7
successfully ended
